cmd/antrea-controller/options.go

```diff
@@ -1,5 +1,4 @@
 // Copyright 2019 Antrea Authors
-//
```

Comment:
unintentional change?
the copyright header is standardized all over the project